### PR TITLE
DIV-6177: Creating flag that identifies a case as server by a process…

### PR DESCRIPTION
**Pull request by jvictorcf**
… server.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: definitions/divorce/json/CaseField/CaseField-alt-service-process-server-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "LiveFrom": "17/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "servedByProcessServer",
```
Comment (madjava) [inline definitions/divorce/json/CaseField/CaseField-alt-service-process-server-nonprod.json:5]: fine by me

**Review by madjava — APPROVED** [review]

File: definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-alt-service-process-server-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "LiveFrom": "17/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "servedByProcessServer",
```
Comment (yarpo) [inline definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-alt-service-process-server-nonprod.json:5]: Field names shouldn't start with uppercase "S"?

Comment (jvictorcf) [reply definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-alt-service-process-server-nonprod.json:5]: Good point. I'll change that now.

Comment (jvictorcf) [reply definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-alt-service-process-server-nonprod.json]: Done

**Review by yarpo — APPROVED** [review]

**Pull request merged**